src/app/containers/time-limited-content-info/time-limited-content-info.component.ts

```diff
@@ -49,10 +51,10 @@ export class TimeLimitedContentInfoComponent {
         if (!submissionUntil) return of(null);
         const timeRemaining = Duration.fromNowUntil(submissionUntil);
         if (!timeRemaining.getMs()) return of(new Duration(0));
-        return interval(1000).pipe(
+        const refreshingRate = timeRemaining.getMs() < 5*60*1000 ? 200 : 1000;
```

Comment:
I prefer to use our regular approach `MINUTES * 5`

Reply:
fixed